test/core/security/grpc_audit_logging_test.cc

```diff
@@ -0,0 +1,106 @@
+//
+//
+// Copyright 2023 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+#include <grpc/support/port_platform.h>
+
+#include <memory>
+
+#include <gtest/gtest.h>
+
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
+
+#include <grpc/grpc_audit_logging.h>
+
+#include "src/core/lib/json/json.h"
+#include "src/core/lib/security/audit_logging/audit_logging.h"
+#include "test/core/util/test_config.h"
+#include "test/core/util/tls_utils.h"
+
+namespace grpc_core {
+namespace testing {
+
+const char kName[] = "test_logger";
+
+using experimental::AuditContext;
+using experimental::AuditLogger;
+using experimental::AuditLoggerFactory;
+using Config = experimental::AuditLoggerFactory::Config;
```

Comment:
I don't think the `Config =` part is needed.

Reply:
IDE warns me that using statements cannot refer to class members. But I removed this and just use `AuditLoggerFactory::Config` subsequently now.